netdata-installer.sh

```diff
@@ -476,20 +476,27 @@ trap build_error EXIT
 # -----------------------------------------------------------------------------
 
 build_libmosquitto() {
+  CFLAGS=
+  LDFLAGS=
+  CXXFLAGS=
+
   if [ "$(uname -s)" = Linux ]; then
     run env CFLAGS= CXXFLAGS= LDFLAGS= make -C "${1}/lib"
   else
     pushd ${1} > /dev/null || return 1
     if [ "$(uname)" = "Darwin" ] && [ -d /usr/local/opt/openssl ]; then
-      run env CFLAGS= CXXFLAGS= LDFLAGS= cmake \
+      LDFLAGS="-L/usr/local/opt/openssl/lib"
+      CXXFLAGS="-I/usr/local/opt/openssl/include"
+
+      run env CFLAGS="$CFLAGS" CXXFLAGS= LDFLAGS= cmake \
```

Comment:
```suggestion
      run env CFLAGS="$CFLAGS" CXXFLAGS= LDFLAGS="$LDFLAGS" cmake \
```

Reply:
To be honest I would replace all `run env` calls with

```
run env CFLAGS="$CFLAGS" CXXFLAGS="$CXXFLAGS" LDFLAGS="$LDFLAGS"
```

and only change the values of CFLAGS, CXXFLAGS and LDFLAGS, in order to keep the logic simpler and increase readability.